InvenTree/InvenTree/exchange.py

```diff
@@ -15,19 +15,19 @@
 class InvenTreeExchange(SimpleExchangeBackend):
     """Backend for automatically updating currency exchange rates.
 
-    Uses the `exchangerate.host` service API
+    Uses the `frankfurter.app` service API
     """
 
-    name = "InvenTreeExchange"
+    name = "InvenTreeExchange-frankfurter"
 
     def __init__(self):
         """Set API url."""
-        self.url = "https://api.exchangerate.host/latest"
+        self.url = "https://api.frankfurter.app/latest"
```

Comment:
Error: spaces in url
Should be:
self.url = "https://api.frankfurter.app/latest"